pkgs/tools/compression/zopfli/default.nix

```diff
@@ -19,6 +19,7 @@ stdenv.mkDerivation rec {
 
   postInstall = ''
     install -Dm444 -t $out/share/doc/zopfli ../README*
+    cp $src/src/zopfli/*.h $dev/include/
```

Comment:
This doesn't actually seem to make any difference?
Note: I change this from `$out` to `$dev` as headers should not be in the runtime closure.

Reply:
@adisbladis right now, the derivation does not include any of these headers: https://github.com/google/zopfli/tree/master/src/zopfli

Reply:
(some of these headers are needed for imagecodecs)

Reply:
My bad, I misread things.